api/app/tests/fba/test_fba_error.feature

```diff
@@ -32,34 +29,59 @@ Feature: /fbc/
         # s_cfb_em :- Spreadsheet Crown Fraction Burned Error Margin
         # s_hfi_em :- Spreadsheet Head Fire Intensity Error Margin
```

Comment:
delete these 2 lines of comments? Doesn't look like the spreadsheet-related stuff is in the test values table anymore